Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStore.cs

```diff
@@ -1235,6 +1235,11 @@ public Task<Result<byte[]>> GetBlobAsync(OperationContext context, ContentHash h
             return GlobalStore.GetBlobAsync(context, hash);
         }
 
+        /// <summary>
+        /// Gets a random machine location, excluding a specified location.
+        /// </summary>
+        public MachineLocation GetRandomMachineLocation(MachineLocation except) => ClusterState.GetRandomMachineLocation(except);

```

Comment:
nit: you may use <inheritdoc /> if you want to avoid comments duplication.

Reply:
This does not inherit from IContentLocationStore

---
In reply to: [312225091](https://github.com/microsoft/BuildXL/pull/705#discussion_r312225091) [](ancestors = 312225091)